module-system/module-implementations/integration-tests/src/chain_state/helpers.rs

```diff
@@ -73,6 +74,22 @@ impl<C: Context, Cond: ValidityCondition> SlotHooks<Cond> for TestRuntime<C, Con
     fn end_slot_hook(&self, _working_set: &mut WorkingSet<<Self::Context as Spec>::Storage>) {}
 }
 
+impl<C: Context, Cond: ValidityCondition> BlobSelector for TestRuntime<C, Cond> {
+    type Context = C;
+
+    fn get_blobs_for_this_slot<'a, I, B>(
+        &self,
+        current_blobs: I,
+        _working_set: &mut WorkingSet<<Self::Context as Spec>::Storage>,
+    ) -> anyhow::Result<Vec<BlobRefOrOwned<'a, B>>>
+    where
+        B: BlobReaderTrait,
+        I: IntoIterator<Item = &'a mut B>,
+    {
+        Ok(current_blobs.into_iter().map(Into::into).collect())
```

Comment:
I think this blob selector should still filter out blobs from unregistered senders, right?

Reply:
It can, but then it will be double work for all honest sequencers.

Sequencer registry has apply_blob hook implemented, where check and all appropriate logic is implemented.

Reply:
Hmm, that's a bit confusing. Can we move that logic into the blob selector?